src/Cards/Finish.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Card, PlayAgainButton, Score, CardWindowBar } from "./styled";
+import close from "../images/close.svg";
+
+const Finish = ({ score }) => {
```

Comment:
The game is finished, so I would renamed this to `Finished` because `Finish` in the present tense sounds like a verb, whereas `Finished` indicates that the user has reached a `finished` state.